pages/listings.tsx

```diff
@@ -1,7 +1,69 @@
-export default function ListingsPage() {
+import AuthGuard from "@App/lib/auth/AuthGuard";
+import useAuthContext from "@App/lib/auth/AuthContext";
+import fetchCurrentUserListings from "@App/lib/listings/useFetchCurrentUserListing";
+import { Card, CardMedia, Stack, CardContent, Box, LinearProgress, Paper, Typography, Button } from "@mui/material";
+import { updateCurrentUser } from "firebase/auth";
+import type { NextPage } from "next";
+import {TListingCollectionReference} from "@App/lib/listings/types"
+
+const ListingsPage: NextPage = () => {
+
+  const { currentUser } = useAuthContext()
+  let obj:TListingCollectionReference;
+  obj={userId: currentUser?.id!}
+  const { data: listings, isLoading } = fetchCurrentUserListings(obj);
+  if (isLoading || !listings) return <LinearProgress />;
+
+  if (listings.empty) return <Typography>No listings found</Typography>;
+
   return (
-    <div>
-      <h1>Listings</h1>
-    </div>
+    <AuthGuard>
+      <Typography variant="h2">My Listings</Typography>
+
+      {listings.docs.map((listing) => (
+        <Box key={listing.id} paddingY={2}>
+          <Card sx={{ display: 'flex' }}>
+            <CardMedia
+              component="img"
+              sx={{ width: 151 }}
+              image="/images/images.jpeg"
+            />
```

Comment:
Seems like the image needs some more style work as it is not completely proportional or responsiv
<img width="1671" alt="Screen Shot 2022-05-12 at 1 10 50 AM" src="https://user-images.githubusercontent.com/56570937/167996452-663e05d0-cb27-4a01-b195-f2e78ba565e5.png">